policy-controller/Cargo.toml

```diff
@@ -20,7 +20,7 @@ clap = { version = "3", default-features = false, features = [
 ] }
 drain = "0.1"
 futures = { version = "0.3", default-features = false }
-k8s-gateway-api = "0.6"
+k8s-gateway-api = "0.6.0"
```

Comment:
?
```suggestion
k8s-gateway-api = "0.6"
```

Reply:
whoops, this is just because i undid the change that removed these deps manually, rather than by `git checkout` or similar.